### PR TITLE
Changes for the use of pytest and stop including tests in code covera…

### PR DESCRIPTION
**Pull request by AndrewTorrance**
## Checklist
  - CHANGELOG.md updated

### CODE REVIEW

**Comment by AndrewTorrance** [comment]
[sdx-validate - implement py.test](https://trello.com/c/0H4MD8FX/496-sdx-validate-implement-pytest)

File: Makefile [hunk 1]
```diff
@@ -4,4 +4,4 @@ build:
 test:
 	pip3 install -r test_requirements.txt
 	flake8 --exclude lib
-	python3 -m unittest tests/*.py
+	pytest -v --cov app
```
Comment (JamesGardiner) [inline Makefile]: Can we have the HTML output locally as well as in Travis? Personally I find it useful for exploring areas that are lacking test coverage.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ### Unreleased
   - Add all service specific config to config file
+  - Change to use pytest to improve test output and code coverage stats
```
Comment (JamesGardiner) [inline CHANGELOG.md]: PyTest doesn't improve our coverage stats

**Review by JamesGardiner — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (AndrewTorrance) [reply CHANGELOG.md]: missing and will change

File: Makefile (hunk 1 above)
Comment (AndrewTorrance) [reply Makefile]: will have a look

**Review by JamesGardiner — DISMISSED** [review]

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**